sunkit_pyvista/tests/test_plotting.py

```diff
@@ -0,0 +1,109 @@
+import os
+import inspect
+import platform
+import warnings
+from pathlib import Path
+
+import pytest
+import pyvista
+from pyvista._vtk import VTK9
+
+import sunpy.data.test as test
+import sunpy.map as smap
+
+from sunkit_pyvista import SunpyPlotter
+
+IMAGE_CACHE_DIR = os.path.join(Path(__file__).parent.absolute(), 'image_cache')
+if not os.path.isdir(IMAGE_CACHE_DIR):
+    os.mkdir(IMAGE_CACHE_DIR)
+# Normal image warning/error thresholds (assumes using use_vtk)
+IMAGE_REGRESSION_ERROR = 500  # major differences
+IMAGE_REGRESSION_WARNING = 200  # minor differences
+
+# Image regression warning/error thresholds for releases after 9.0.1
+HIGH_VARIANCE_TESTS = {}
+VER_IMAGE_REGRESSION_ERROR = 1000
+VER_IMAGE_REGRESSION_WARNING = 1000
+
+glb_reset_image_cache, glb_ignore_image_cache = False, False
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Don't we want this in the conftest.py file so we can use this across the library?

Reply:
the global variables won't be accessible in test_plotting then right?

Reply:
But what if we want to have figure tests in other files? Or is the plan to only support it in this file?

Reply:
We can just call that fixture there right? For now all figure tests are only local to this file

Reply:
Not sure.

Reply:
> the global variables won't be accessible in test_plotting then right?
> We'd have to use them in every function individually

If you put them in `conftest.py`, `pytest` will make them available in test files, so I think we should have a `conftest.py` file with everything from line 94 up in it.

Reply:
This won't really work the way pyvista intended it to, because verify_image_cache is set as a callback, making this a fixture won' really work right?